File: js/lc1431.js
var kidsWithCandies = (candies, extraCandies)=>{
    let maximum = candies.reduce((a, b)=> Math.max(a, b));
    let ans = [];
    let n = candies.length;
    for(let i=0;i<n;i++){
        ans.push((maximum<=(candies[i]+extraCandies))?true:false);
    }
    return ans;
};

File: py/lc1431.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        maximum = max(candies)
        res = []
        for i in range(len(candies)):
            if maximum<=(candies[i]+extraCandies):
                res.append(True)
            else:
                res.append(False)
        return res
        